en/db-models-relationships.md

```diff
@@ -1363,4 +1363,4 @@ $customer->getInvoices()->delete(
 
 [db-normalization]: https://en.wikipedia.org/wiki/Database_normalization
```

Comment:
Shouldn't you add this one up there like so?

[Database normalization] [db-normalization]

(without the space)